src/components/Table/Table.js

```diff
@@ -18,7 +21,13 @@ class Table extends Component {
         return new Error('`Table` must only have `TableColumn` children.');
       }
     },
+    compact: PropTypes.bool,
     data: PropTypes.array.isRequired,
+    fixed: PropTypes.bool,
+    selectable: PropTypes.bool,
+    size: PropTypes.oneOf(_.values(SIZES)),
+    striped: PropTypes.bool,
+    very: PropTypes.bool,
   };
 
```

Comment:
break out all table styles into bool props, or strings, based on the class name.